app/views/main.scala.html

```diff
@@ -55,6 +55,13 @@
         seq
     }
 
+@gaCustomEvent = @{
```

Comment:
gaCustomEvent is never used, consider adding it to inlineScript

Reply:
done